src/ngraph/runtime/interpreter/int_backend.hpp

```diff
@@ -24,10 +24,6 @@
 
 #include "ngraph/runtime/tensor.hpp"
 
-#ifdef NGRAPH_DISTRIBUTED_ENABLE
-#include "ngraph/runtime/reference/allreduce.hpp"
```

Comment:
Any reason for this action ?

Reply:
This file does not need to include any distributed header files.